packages/macro/src/index.ts

```diff
@@ -49,9 +27,28 @@ const jsMacroTags = new Set([
 
 const jsxMacroTags = new Set(["Trans", "Plural", "Select", "SelectOrdinal"])
 
+let config: LinguiConfigNormalized
+
+function getConfig(_config?: LinguiConfigNormalized) {
+  if (_config) {
+    config = _config
+  }
+  if (!config) {
+    config = loadConfig({ configPath: process.env.LINGUI_CONFIG })
+  }
```

Comment:
if config passed directly to macro - use it
if no, use try to load config. Then store it in global variable to avoid loading it on every macro call